rogue-thi-app/.gitignore

```diff
@@ -1,5 +1,3 @@
-android/
```

Comment:
i dislike the idea of having a bunch of auto generated files in the repo. We already went though great lengths removing the APO json file and the PWA icons and auto generate them in the docker file.

Are all files in `android/` auto generated using capacitor? Or are there custom modifications?
In the former case we can just remove the foler? In the latter case i guess the best option is to move the folder into a seperate repository and include that as a submodule too?

Reply:
Personally I prefer having everything checked into a single repository, it avoids complexity and allows us to modify the files if necessary

Reply:
As you can see in the `.gitmodules` file, I've already added the Android repository as submodule. The question is whether we want to replace the current `android` directory, since the submodule currently resides in a directory called `android-repo`.

Reply:
+1 for moving the capacitor generated files into that repo